Formula/m/miniupnpc.rb

```diff
@@ -2,6 +2,7 @@ class Miniupnpc < Formula
   desc "UPnP IGD client library and daemon"
   homepage "https://miniupnp.tuxfamily.org"
   url "https://miniupnp.tuxfamily.org/files/download.php?file=miniupnpc-2.2.6.tar.gz"
+  mirror "http://miniupnp.free.fr/files/download.php?file=miniupnpc-2.2.6.tar.gz"
```

Comment:
let's maybe use the debian one, http://ftp.debian.org/debian/pool/main/m/miniupnpc/miniupnpc_2.2.6.orig.tar.gz